public/views/index-template.html

```diff
@@ -260,6 +254,7 @@
         var preloader = document.getElementsByClassName("preloader");
         if (preloader.length) {
           preloader[0].className = "preloader preloader--done";
+          window.parent.postMessage('iframeLoaded', window.parent.location.origin);
```

Comment:
Is this for the loading spinner? Or is this PR intending to resurrect your `postMessage` changes as well?

Reply:
Loading spinner - added a comment.